src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -35,6 +35,23 @@ namespace winrt::Microsoft::Terminal::Core
 namespace Microsoft::Terminal::Core
 {
     class Terminal;
+
+    enum class InternalSelectionDirection

```

Comment:
we decided to go with an internal copy of the winrt enum instead of just [including `Microsoft.Terminal.Core` in the WPF Control](https://github.com/microsoft/terminal/commit/ad9900857aa44b3ccbde148cc5ecb53d1a2e5b18) - any reason why?

Reply:
Notes left on commit as to why ;)

Reply:
okay fine yea that felt dirty to write, but I figured the PublicTerminalCore was a. already dirty and b. long overdue for a big refactor that would get rid of the hack anyways